process-engine/bpmn-studio
# :sparkles: Unify Exports

## What did you change?

Previously when exporting the diagram as bpmn the last cached xml was exported.
Now when exporting the diagram as bpmn the current xml gets exported.

- Unifies the exports

## How can others test the changes?

- Open up the BPMN-Studio 
- Go to the Detail View of any diagram 
- Add changes to the diagram (Do not save the diagram)
- Export the diagram as bpmn
- Import the exported bpmn
- Notice that the imported bpmn has the changes

## PR-Checklist

Please check the boxes in this list after submitting your PR:
